tests/test_vision_explanation.py

```diff
@@ -196,6 +197,43 @@ def test_vision_explain_loadmodel():
         os.remove(elt)
 
 
+def test_pointing_game():
```

Comment:
should we parameterize this test for different thresholds/num masks or would that be too much?